### ISSUE TITLE
[Accessibility Audit] Update Alt-text for VA logo / Seal

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#75943]
The forms-audit-digitize team received a 508 audit report back on a recent form we shipped. In it, they identified an issue with the logo on the modal sign-in window for logging in.

The report stated
> WCAG 1.1.1 Non-text Content | All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below.
> • On the Sign in dialog screen (Sign in to start your request Button), the text equivalent for the VA logo and Seal, U.S. Department of Veterans Affairs image is not sufficient. The alt text should be "U.S Department of Veterans Affairs Seal" but is "[va.gov](http://va.gov/)". (New: 02/06/2024)

After looking at the alt text on va.gov for the same image, my recommendation is that the alt text should actually read

> VA logo and Seal, U.S. Department of Veterans Affairs

I've attached a PDF of the audit report (There's also another issue in it for my team, yours is the one marked as `Defect 1`
[40627 va.gov - Request personal records (VA Form 20-10206)  2_06_2024 - 508 Audit Report - 2_12_2024 1_27 PM.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/14267762/40627.va.gov.-.Request.personal.records.VA.Form.20-10206.2_06_2024.-.508.Audit.Report.-.2_12_2024.1_27.PM.pdf)

I also [talked with the team on Slack](https://dsva.slack.com/archives/CSFV4QTKN/p1707778250930219) about this as well.
